components/prism-elm.js

```diff
@@ -3,33 +3,45 @@ Prism.languages.elm = {
 		pattern: /(^|[^-!#$%*+=?&@|~.:<>^\\\/])(--[^-!#$%*+=?&@|~.:<>^\\\/].*|{-[\s\S]*?-})/m,
 		lookbehind: true
 	},
-	char: /'([^\\']|\\([abfnrtv\\"'&]|\^[A-Z@[\]\^_]|NUL|SOH|STX|ETX|EOT|ENQ|ACK|BEL|BS|HT|LF|VT|FF|CR|SO|SI|DLE|DC1|DC2|DC3|DC4|NAK|SYN|ETB|CAN|EM|SUB|ESC|FS|GS|RS|US|SP|DEL|\d+|o[0-7]+|x[0-9a-fA-F]+))'/,
-	string: {
-		pattern: /"([^\\"]|\\([abfnrtv\\"'&]|\^[A-Z@[\]\^_]|NUL|SOH|STX|ETX|EOT|ENQ|ACK|BEL|BS|HT|LF|VT|FF|CR|SO|SI|DLE|DC1|DC2|DC3|DC4|NAK|SYN|ETB|CAN|EM|SUB|ESC|FS|GS|RS|US|SP|DEL|\d+|o[0-7]+|x[0-9a-fA-F]+)|\\\s+\\)*"/,
+	char: {
+		pattern: /'([^\\'\r\n]|\\([abfnrtv\\']|\d+|x[0-9a-fA-F]+))'/,
 		greedy: true
 	},
+	string: [
+		{
+			// Multiline strings are wrapped in triple ". Quotes may appear unescaped.
+			pattern: /"""((.|\s)*?(?="""))"""/,
```

Comment:
Is there an actual need for the positive look-ahead here?
Also, can you rewrite `(.|\s)` as `[\s\S]`? It's what we use everywhere else as a match-everything token.